devel/sbt/files/sbt.sh

```diff
@@ -25,7 +25,9 @@ fi
 if [ -z "$JAVA_OPTS" ]; then
     # Ensure enough heap space is created for sbt.  These settings are
     # the default settings from Typesafe's sbt wrapper.
-    JAVA_OPTS="-XX:+CMSClassUnloadingEnabled -Xms1536m -Xmx1536m -XX:MaxPermSize=384m -XX:ReservedCodeCacheSize=192m -Dfile.encoding=UTF8"
+    JAVA_VERSION=$(java -version 2>&1 | awk -F '"' '/version/ {print $2}')
+    CLASS_METADATA_OPT=$([ "$JAVA_VERSION" \< "1.8" ] && echo "MaxPermSize" || echo "MaxMetaspaceSize")
```

Comment:
Can you switch this to standard if/else/fi syntax please then it's good to merge.

Reply:
Thanks. Done.